src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs

```diff
@@ -96,15 +98,15 @@ public NotificationOperations(ILogTracer log, IOnefuzzContext context)
         return null;
     }
 
-    private void GithubIssue(GithubIssuesTemplate config, Container container, string filename, RegressionReportOrReport? report) {
+    private void GithubIssue(GithubIssuesTemplate config, Container container, string filename, IReport? report) {
```

Comment:
can we make the report parameter non nullable. this will save us a null check in the implementation since we check for null before the call. (same goes for NotifyTeams)